DMTN-199.tex

```diff
@@ -29,7 +29,7 @@
 \setDocRef{DMTN-199}
 \setDocUpstreamLocation{\url{https://github.com/lsst-dm/dmtn-199}}
 
-\date{2024-09-16}
+\date{2025-05-14}
```

Comment:
you can switch to \today while we draft. I will 'fix' the date when we tag the new version. I guess we may need to update this .. https://developer.lsst.io/project-docs/change-controlled-docs.html#releasing-a-new-version-from-the-main-branch